bin/pps2018_runner.py

```diff
@@ -180,11 +182,18 @@ def pps_worker(scene, publish_q, input_msg, options):
             err_reader2.join()
 
         pps_control_path = my_env.get('STATISTICS_DIR', options.get('pps_statistics_dir', './'))
-        xml_files = create_xml_timestat_from_ascii(scene, pps_control_path)
-        xml_files = xml_files + get_product_statistics_files(pps_control_path,
-                                                             scene,
-                                                             options['product_statistics_filename'],
-                                                             options['pps_filetime_search_minutes'])
+        if "file4pps" in scene:
```

Comment:
This `if` statement is repeated in `create_xml_timestat_from_lvl1c` and `create_product_statistics_from_lvl1c`, could these function take care of calling the right function instead?

Reply:
I was thinking that the old functions: create_xml_timestat_from_ascii and get_product_statistics_files can "soon" be removed. And for that I wanted totally separate functions in the utils.py. It is the same statement but with different meanings. Here it checks if we have version 2018 or version 2021 and in the utils.py the check handles bad input. Currently the check in utils.py is not needed as we have the check here. But when we remove the check here the functions in utils.py should have the check.

Reply:
Ok, I understand. I'm ok with leaving the logic as it is now. However, would it be possible to change that `if` statement with something more explicit? For example here, could the version of  acpg be checked directly? and in the called function, could we maybe have a try/except instead to make it clear it happens when something goes wrong?